syntaxes/odoc.json

```diff
@@ -0,0 +1,81 @@
+{
+  "scopeName": "source.ocaml.odoc",
+  "name": "comment.other.odoc",
+  "fileTypes": ["mld"],
+  "patterns": [
+    {
+      "name": "comment.other.odoc",
+      "begin": "",
+      "end": "",
+      "patterns": [{ "include": "#markup" }]
+    },
+    {
+      "comment": "section header",
+      "begin": "\\{\\s\\{[0-9]\\}\\s[[\\w+\\s+]+[\\W]]*\\b",
```

Comment:
In the documentation `{ 0 … 9 }+` means one or more digits.

Instead of `\\{[0-9]\\}`, you should do `[[:digit:]]+`.